x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/searchable_snapshot_field/searchable_snapshot_field.tsx

```diff
@@ -0,0 +1,231 @@
+  const { isUsingSearchableSnapshotInHotPhase } = useSearchableSnapshotState();
+  const searchableSnapshotPath = `phases.${phase}.actions.searchable_snapshot.snapshot_repository`;
+  const { isLoading, error, data, resendRequest } = useLoadSnapshotRepositories();
+
+  const repos = data?.repositories ?? [];
+
+  let calloutContent: React.ReactNode | undefined;
+
+  if (!isLoading) {
+    if (error) {
+      calloutContent = (
+        <FieldLoadingError
+          resendRequest={resendRequest}
+          data-test-subj="repositoriesErrorCallout"
+          aria-label={i18n.translate(
+            'xpack.indexLifecycleMgmt.editPolicy.reloadSnapshotRepositoriesLabel',
+            {
+              defaultMessage: 'Reload snapshot repositories',
+            }
+          )}
+          title={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.editPolicy.loadSnapshotRepositoriesErrorTitle"
+              defaultMessage="Unable to load snapshot repositories"
+            />
+          }
+          body={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.editPolicy.loadSnapshotRepositoriesErrorBody"
+              defaultMessage="Refresh this field and enter the name of an existing snapshot repository."
+            />
+          }
+        />
+      );
+    } else if (repos.length === 0) {
+      calloutContent = (
+        <EuiCallOut
+          data-test-subj="noSnapshotRepositoriesCallout"
+          iconType="help"
+          color="warning"
+          title={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.editPolicy.noSnapshotRepositoriesFoundTitle"
+              defaultMessage="No snapshot repositories found"
+            />
+          }
+        >
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.noSnapshotRepositoriesFoundMessage"
+            defaultMessage="{link} to use searchable snapshots."
+            values={{
+              link: (
+                <EuiLink
+                  href={getUrlForApp('management', {
+                    path: `data/snapshot_restore/add_repository`,
+                  })}
+                  target="_blank"
+                >
+                  {i18n.translate(
+                    'xpack.indexLifecycleMgmt.editPolicy.createSearchableSnapshotLink',
+                    {
+                      defaultMessage: 'Create a snapshot repository',
+                    }
+                  )}
+                </EuiLink>
+              ),
+            }}
+          />
+        </EuiCallOut>
+      );
+    }
+  }
+
+  return (
+    <DescribedFormField
+      data-test-subj={`searchableSnapshotField-${phase}`}
+      switchProps={{
+        'data-test-subj': 'searchableSnapshotToggle',
+        label: i18n.translate(
+          'xpack.indexLifecycleMgmt.editPolicy.searchableSnapshotsToggleLabel',
+          { defaultMessage: 'Use searchable snapshot' }
```

Comment:
```suggestion
          { defaultMessage: 'Create searchable snapshot' }
```

Reply:
Users are creating and mounting the searchable snapshot, so I think that **Create** is more accurate here than **Use**.